desdeo/problem/testproblems.py

```diff
@@ -1045,6 +1045,96 @@ def simple_scenario_test_problem():
         scenario_keys=["s_1", "s_2"],
     )
 
+def re21() -> Problem:
+    r"""Defines the four bar truss design problem.
+
+    The objective functions and constraints for the four bar truss design problem are defined as follows:
+
+    \begin{align}
+        &\min_{\mathbf{x}} & f_1(\mathbf{x}) & = L(2x_1 + \sqrt{2}x_2 + \sqrt{x_3} + x_4) \\
+        &\min_{\mathbf{x}} & f_2(\mathbf{x}) & = \frac{FL}{E}\left(\frac{2}{x_1} + \frac{2\sqrt{2}}{x_2}
+        - \frac{2\sqrt{2}}{x_3} + \frac{2}{x_4}\right) \\
+        &\text{s.t.,}   & \frac{F}{\sigma} \leq x_1 & \leq 3\frac{F}{\sigma},\\
+        & & \sqrt{2}\frac{F}{\sigma} \leq x_2 & \leq 3\frac{F}{\sigma},\\
+        & & \sqrt{2}\frac{F}{\sigma} \leq x_3 & \leq 3\frac{F}{\sigma},\\
+        & & \frac{F}{\sigma} \leq x_4 & \leq 3\frac{F}{\sigma}.\\
+    \end{align}
+
+    References:
+        Cheng, F. Y., & Li, X. S. (1999). Generalized center method for multiobjective engineering optimization.
+            Engineering Optimization, 31(5), 641-661.
+
+        Tanabe, R. & Ishibuchi, H. (2020). An easy-to-use real-world multi-objective
+            optimization problem suite. Applied soft computing, 89, 106078.
+            https://doi.org/10.1016/j.asoc.2020.106078.
+
+        https://github.com/ryojitanabe/reproblems/blob/master/reproblem_python_ver/reproblem.py
+
+    Returns:
+        Problem: an instance of the four bar truss design problem.
+    """
+    # should these be hardcoded here or given as optional arguments for the problem?
```

Comment:
Add these as optional arguments with default values set to the current values given to them.